pkg/apis/pipeline/v1alpha1/workspace_types.go

```diff
@@ -24,5 +24,4 @@ import (
 type WorkspaceDeclaration = v1alpha2.WorkspaceDeclaration
 
 // WorkspaceBinding maps a Task's declared workspace to a Volume.
-// Currently we only support PersistentVolumeClaims, EmptyDir and ConfigMap.
```

Comment:
Removed as suggested in https://github.com/tektoncd/pipeline/pull/1800#discussion_r366454331